mkdocs.yml

```
@@ -51,12 +51,11 @@ theme:
     - yaml
     - rust
     - django
+  analytics:
+    gtag: G-1851JH9FSR
```

```suggestion
    gtag: "G-1851JH9FSR"
```

not sure what's going on, but doing this sometimes fixes yaml bugs